Dockerfile.patch

```diff
@@ -0,0 +1,11 @@
+--- Dockerfile	2020-08-27 14:25:21.934595772 -0400
++++ Dockerfile.local	2020-08-27 14:25:14.753559120 -0400
+@@ -24,6 +24,8 @@
+       openssl \
+       rsync \
+       " && \
++    rm /etc/yum.repos.d/* && \
++    curl -o /etc/yum.repos.d/art.repo https://gist.githubusercontent.com/jupierce/4b6eeef00e45f377c2ecb6ceb4d4a4fc/raw/271643df5471b76f75f0b50243b31be202f75a7d/art.repo && \
```

Comment:
Consider conserving the `art.repo` file in our repo. We should stay able to work locally if GH is down.